### PR TITLE
SE-3151: cashflow with notional and interest

### PR DESCRIPTION
**Pull request by liberty-askew**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [ ] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR
Update notebook with Cashflow broken down into Notional and Interest. 
Should fix failing tests in LUSID pipeline.
Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: examples/use-cases/instruments/Term Deposit.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
Comment (rizwansaeed) [inline examples/use-cases/instruments/Term Deposit.ipynb:2]: <p>add code formatting to <code>get_upsertable_portfolio_cash_flow</code></p><p> </p><p>i think we should also define CTVoM (Constant Time Value of Money) as we havent said what it is anywhere and new reader might not know straight away</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/699/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/Term Deposit.ipynb:2]: <p>just to confirm, <code>Notional</code> and <code>Interest</code> arent in our default set oftxn types?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/699/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/Term Deposit.ipynb:2]: <p>remove extra "$" (I presume its a typo)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/699/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (liberty-askew) [reply examples/use-cases/instruments/Term Deposit.ipynb:2]: <p>Have defined CTVoM in section 5 with a link to the KB - let me know if you want more detials</p>

**Pull request merged**